pkg/metrics/service-monitor.go

```diff
@@ -50,3 +83,12 @@ func GenerateServiceMonitor(s *v1.Service) *monitoringv1.ServiceMonitor {
 		},
 	}
 }
+
+// IsServiceMonitor checks if ServiceMonitor is registered in the cluster.
+func IsServiceMonitor(config *rest.Config) (bool, error) {
```

Comment:
Also, a nit: the name `IsServiceMonitor` could be confusing. Maybe `HasServiceMonitor` or `IsServiceMonitorRegistered`?

Reply:
`HasServiceMonitor` sounds good, will also unexport it as agreed in the above comment.